ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml

```diff
@@ -0,0 +1,77 @@
+---
+- set_fact:
+    my_user: "{{ item }}"
+
+- name: per_user {{my_user}} Tasks Started
+  debug:
+    msg: "per_user {{my_user}} Tasks - Started"
+
+# Create user project and make him admin
+- name: Create workshop user's project for user "{{ my_user }}"
+  shell: oc adm new-project "{{ my_user }}"
```

Comment:
you don't need the shell module most of the time, command is enough:

```
command: oc adm new-project "{{ my_user }}"
```

Use shell only when you need Shell features: pipes, input/output redirects, subshell, or multiple commands, etc.